namer/metadataapi.py

```diff
@@ -188,7 +188,8 @@ def __match_weight(result: ComparisonResult) -> float:
     value = 0.00
     if result.phash_distance is not None:
```

Comment:
notice we're already in a phash block here..... and if the value can't reach 1000 for a perfect phash match, these will always be behind a name match, and possibly name failures.....

Reply:
okay, maybe this 70e821f

Reply:
sure, worth a shot